api/envoy/extensions/filters/http/aws_lambda/v3/aws_lambda.proto

```diff
@@ -55,6 +56,12 @@ message Config {
   // Changing the value of the host header can result in a different route to be selected
   // if an HTTP filter after AWS lambda re-evaluates the route (clears route cache).
   string host_rewrite = 4;
+
+  // Specifies the credentials profile to be used from the AWS credentials file.
+  // This parameter is optional. If set, it will override value set in AWS_PROFILE env variable and
```

Comment:
nit: s/override value set in AWS_PROFILE/override the value set in the AWS_PROFILE